exchanges/bittrex/bittrex_types.go

```diff
@@ -76,6 +76,7 @@ type TickerData struct {
 	LastTradeRate float64 `json:"lastTradeRate,string"`
 	BidRate       float64 `json:"bidRate,string"`
 	AskRate       float64 `json:"askRate,string"`
+	UpdatedAt     time.Time
```

Comment:
👁️

Reply:
I'm not sure what you're meaning by this. I'm guessing you'd like a `json` field, though this does automatically get populated, but I'll add it